### ISSUE TITLE
Can't navigate into clips from the clips sidebar using keyboard navigation

### ISSUE THREAD

**Issue by robby1066** [robby1066/keepposted-help#29]
# Describe the bug
When trying to record a message using just the keyboard, it's not possible to drill into a clip from the list of clips in the sidebar. It's possible to delete a clip, but there is no way to focus the clip item that will allow you to tab into it.

## Steps To Reproduce
1. Go to any message page from the 'drafts' page (or create a new message and go to the page where you would record the clips)
2. Using the keyboard, tab through until you reach the sidebar.
3. Observe that it's impossible to focus on the clips in the sidebar and press `Enter` to navigate directly to that clip.

## Expected behavior
The clips should be accessible from the list when the person is using keyboard-only navigation

## Screenshots
![image](https://user-images.githubusercontent.com/342510/106803578-f977c380-6629-11eb-9a59-81f941e85f80.png)

## Additional context
It's still technically possible to record clips by skipping over the sidebar entirely and using the linear navigation (continue button), but it's cumbersome.

**Comment by robby1066** [comment]
https://user-images.githubusercontent.com/342510/107127320-f3c4ec80-687a-11eb-80d4-1aba515a7b62.mp4

**Comment by robby1066** [comment]
Deployed and tested on Feb 8, 2021
